difficult_stage_1/main.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style3.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body onload='loadPage()'>
+    </script>
+    <h1 class = 'white' style='font-size: 50px;'>Lets guess</h1> 
+    <div class = 'key-input'>
+      <div class = 'key-input-group d-block text-center'>
+        <div id='qn'class = 'white' style='font-size: 20px;'></div>
+        <br/>
+        <div>
+          <div id='audio'>
+          </div>
+        </div>
+        <br/>
+        <label for='ans'>Answer:</label>
+        <input text='Ans' class='letter' id='input' name='input' required />
+      </div>
+    </div>
+    <br/>
+    <div class="text-center">
+      <button type="submit" id='submit' class='submit'>SUBMIT</button>
+    </div>
+    
+    <script src='script.js'>
+    </script>
+  </body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script>
+    const audioSequence = [{qn:"which country was Alan Turing born in?", audioClip:"source/qn1 (1).mp3"}, {qn:"When did Bill Gates make the first computer?", audioClip:"source/qn2 (1).mp3"}, {qn:"What is the first mobile App in the world?", audioClip:"source/qn3 (1).mp3"},{qn:"What is the most popular computer language now?", audioClip:"source/qn4 (1).mp3"}, {qn:"What were Steve Jobs last designs?", audioClip:"source/qn5 (1).mp3"}]
+    let index = 0
+
+    function loadPage(){
+
+      //display audio
+      audioDisplay()
+      //display qn
+      qn = document.getElementById('qn')
+      qn.textContent = audioSequence[index].qn
+
+      console.log(index)
+    }
+
+    function audioDisplay() {
+      container = document.getElementById('audio')
+      var x = document.createElement('AUDIO')
+      x.canPlayType = 'audio/mpeg'
+      x.setAttribute('src', audioSequence[index].audioClip)
+      x.setAttribute('controls', 'controls')
+      container.appendChild(x)
+    }
+
+    function sent() {
+      ansInput = document.getElementById('input').value
+  //im not sure whether this one works, and i hvn't added the thing can post img from the server side 
+  //please check and i will change accordingly 
+      $.post("https://stem-week-cipher.herokuapp.com", [ansInput])
```

Comment:
You will need to change the URL to `https://stem-week-cipher.herokuapp.com/testPicture` for a picture. POST requesting to the root route won't give you back anything as I didnt configure the route to do anything

Also I dont really understand what is the `[ansInput]` part for. Mind to clarify?

Reply:
erm, i check the jquery post() it says that I needa put the data that I wanna send to the server end in the [], so the ansInput would be the input value.. I'm tbh also not quite sure that's why I commented there

Reply:
Check discord for implementation way as well as the structure of the input (in the form of JSON) to the server. Though if I remember correctly, the output for your code is an image (of a QR code) not a JSON. hence. some experimentation is needed and you cannot just copy directly off the code in discord :) (in terms of the either `success` handler of jQuery Ajax or the `.then()` functions of the fetch API)